cmd/descheduler/app/options/options.go

```diff
@@ -89,6 +89,7 @@ func (rs *DeschedulerServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&rs.NodeSelector, "node-selector", rs.NodeSelector, "DEPRECATED: selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
 	// max-no-pods-to-evict limits the maximum number of pods to be evicted per node by descheduler.
 	fs.IntVar(&rs.MaxNoOfPodsToEvictPerNode, "max-pods-to-evict-per-node", rs.MaxNoOfPodsToEvictPerNode, "DEPRECATED: limits the maximum number of pods to be evicted per node by descheduler")
+	fs.IntVar(&rs.MaxNoOfPodsToEvictPerNamespace, "max-pods-to-evict-per-namespace", rs.MaxNoOfPodsToEvictPerNamespace, "limits the maximum number of pods to be evicted per namespace by descheduler")
```

Comment:
I don't think we need to add a flag for this. We were migrating away from flags to the component config API

Reply:
Hi . Sorry for I don't notice this. Is it the MR: https://github.com/kubernetes-sigs/descheduler/pull/587 for migrate away from flags?
Is it wait this MR done first will be better?

Reply:
Please see https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/descheduler/descheduler.go#L89-L115 and https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/apis/componentconfig/types.go#L47-L48. We prefer the policy config since it's versioned.

Reply:
Thanks very much~~ I will check and fix it as soon as possible.

Reply:
Fix done, please check~~